utoipa-swagger-ui/Cargo.toml

```
@@ -49,7 +49,7 @@ no-default-features = true
 rustdoc-args = ["--cfg", "doc_cfg"]
 
 [build-dependencies]
-zip = { version = "2.6", default-features = false, features = ["deflate"] }
+zip = { version = "3.0.0", default-features = false, features = ["deflate"] }
```

Why not pinning it to:
```suggestion
zip = { version = "3.0", default-features = false, features = ["deflate"] }
```